docs/reference/requirement-sets/add-in-commands-requirement-sets.md

```diff
@@ -16,7 +16,9 @@ The following table lists the add-in commands 1.1 requirement set, the Office ho
 
 |  Requirement set  |  Office 2013 for Windows | Office 2016 for Windows (non-subscription) | Office 365 for Windows   |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  |  
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-| AddinCommands 1.1  | N/A | 16.0.4678.1000 *Supported in Outlook only*  | Version 1705 (Build 8121.1000) or later | N/A | 15.34 or later| May 2017 | |
+| AddinCommands 1.1  | N/A | 16.0.4678.1000 *Supported in Outlook only*  | Version 1705 (Build 8121.1000) or later | N/A | 15.34 or later\*| May 2017 | |
+
+>\* The [Office.context.requirements.isSetSupported](https://docs.microsoft.com/javascript/api/office/office.requirementsetsupport?view=office-js#issetsupported-name--minversion-) method will erroneously return `false` for versions 16.9 &ndash; 16.14 (inclusive), but the requirement set *is* supported on these versions.
```

Comment:
Is there a reason you're not tagging this as [!NOTE] or one of the other notations? Just curious.